src/test/java/org/magicdgs/readtools/tools/conversion/StandardizeReadsIntegrationTest.java

```diff
@@ -42,39 +42,195 @@
-                        expectedPaired},
-                {"StandardizeReads_single_SAM",
+                                .addArgument("rawBarcodeSequenceTags", "B2")
+                                .addInput(TestResourcesUtils.getWalkthroughDataFile("bc_in_two_tags.dual_index.SE.sam")),
+                        STANDARD_DUAL_INDEX_SE},
+                {"bc_in_two_tags.dual_index.paired",
                         new ArgumentsBuilder()
-                                .addInput(getTestFile("small.single.sam"))
                                 .addArgument("rawBarcodeSequenceTags", "BC")
-                                .addArgument("rawBarcodeSequenceTags", "B2"),
-                        expectedSingle},
-                {"StandardizeReads_paired_FASTQ",
-                        new ArgumentsBuilder()
-                                .addInput(TestResourcesUtils.getExampleDataFile("SRR1931701.illumina_1.fq"))
-                                .addFileArgument("input2", TestResourcesUtils.getExampleDataFile("SRR1931701.illumina_2.fq")),
-                        expectedPaired},
-                {"StandardizeReads_single_FASTQ",
-                        new ArgumentsBuilder()
-                                .addInput(TestResourcesUtils.getExampleDataFile("SRR1931701.illumina_se.fq")),
-                        expectedSingle},
-                {"StandardizeReads_single_SAM_names",
-                        new ArgumentsBuilder()
-                                .addInput(getTestFile("small.single.name.sam"))
-                                .addBooleanArgument("barcodeInReadName", true),
-                        expectedSingle},
+                                .addArgument("rawBarcodeSequenceTags", "B2")
+                                .addBooleanArgument("interleaved", true)
+                                .addInput(TestResourcesUtils.getWalkthroughDataFile("bc_in_two_tags.dual_index.paired.sam")),
+                        STANDARD_DUAL_INDEX_PAIRED},
+
+                // standardize FASTQ files
+                // with Casava read names
+                {"casava.single_index.paired",
+                        new ArgumentsBuilder()
+                                .addInput(TestResourcesUtils.getWalkthroughDataFile("casava.single_index.paired_1.fq"))
+                                .addFileArgument("input2", TestResourcesUtils.getWalkthroughDataFile("casava.single_index.paired_2.fq")),
+                        STANDARD_SINGLE_INDEX_PAIRED},
+                // TODO: with Casava format, single-end data is marked as first of pair
+                // TODO: and we should force somehow single-end processing to set the unpaired flag
+                // TODO: thus, this isn't in the expected output (should be marked as single-end)
+                // TODO: this is related with https://github.com/magicDGS/ReadTools/issues/118
+                // TODO: to enable this test, this should be solved
```

Comment:
Open a new issue (#306) because it is related, but not really the same. Should be mentioned here too.